src/core/qgspluginlayerregistry.cpp

```diff
@@ -57,10 +57,10 @@ QgsPluginLayerRegistry::~QgsPluginLayerRegistry()
   if ( !mPluginLayerTypes.isEmpty() )
```

Comment:
Isn't this check redundant?

Reply:
Indeed.

Reply:
I've got my finger ready on the merge button, better remove this statement before the button's clicked ;)

Reply:
Heh. I'm not anywhere close to my workstation until tomorrow morning. Feel free to add a code suggestion I can accept, or lift your finger for 14 hours ;)

Reply:
3 suggestions for a single change seems a lot. I've convinced my finger to snooze for now